polly/plugins/maya/publish/submit_deadline.py

```diff
@@ -25,7 +25,8 @@ def process(self, instance):
         from avalon import api
         from avalon.vendor import requests
 
-        assert api.Session["AVALON_DEADLINE"], "Requires AVALON_DEADLINE"
+        deadline = api.Session.get("AVALON_DEADLINE", None)
```

Comment:
* **could we make the variable name AVALON_DEADLINE.**
It will improve the the reading so why not.

* **we could skip the .get() and assert and just attempt to get the key directly**.
something along the line of `assert "AVALON_DEADLINE" in api.Session, "Improved error message here"` would be better. More explicit but simple and elegant ;)